source/adapters/native_cpu/device.cpp

```diff
@@ -60,7 +60,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(ur_device_handle_t hDevice,
   UR_ASSERT(hDevice, UR_RESULT_ERROR_INVALID_NULL_HANDLE);
   UrReturnHelper ReturnValue(propSize, pPropValue, pPropSizeRet);
 
-  switch (propName) {
+  switch (static_cast<uint32_t>(propName)) {
```

Comment:
I understand that this is how the rest of the adapters suppress this problem, but wouldn't it be better to add something like:
```
  default:
     warn("unimplemented query {}", propName);
     break;
```